.python-version

```diff
@@ -0,0 +1 @@
+3.8.10
```

Comment:
This doesn't match the point release in python-docker's .python-version, is that expected?

Reply:
It isn't, but I think that python-docker's .python-version is at fault.

```sh
opensafely exec python --version
```

returns `Python 3.8.10`.